docs/website/package.json

```diff
@@ -46,26 +45,55 @@
     "clean": {
       "command": "rimraf dist"
     },
+    "lint": {
+      "dependencies": [
+        "lint:scss",
+        "lint:ts"
+      ]
+    },
+    "lint:scss": {
+      "command": "stylelint \"src/**/*.scss\" --color",
```

Comment:
maybe `"command": "stylelint 'src/**/*.scss' --color",` will be fine? Or we need to ascape it and have `"`?